### PR TITLE
Changing example topic and subscription names to my-topic and my-sub …

### PR DESCRIPTION
**Pull request by kenfesta**
…to establish conventions across Pub/Sub

Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by JustinBeckwith** [comment]
@fhinkel @ace-n not sure where this is comin from :) Is this something we're discussing at a high level across all samples?

**Comment by fhinkel** [comment]
Not that I'm aware of.

File: samples/quickstart.js [hunk 1]
```diff
@@ -20,15 +20,15 @@
 const PubSub = require('@google-cloud/pubsub');
 
 // Your Google Cloud Platform project ID
-const projectId = 'YOUR_PROJECT_ID';
```
Comment (fhinkel) [inline samples/quickstart.js]: So AFAIK all projects use 'YOUR_PROJECT_ID'. I'd prefer to keep it consistent over projects.

**Comment by fhinkel** [comment]
Do you want to have a look at the errors from the linter?

**Pull request merged**